Makefile

```diff
@@ -185,3 +153,78 @@ clean:
 .PHONY: nuke
 nuke: clean
 	sudo rm -rf $(shell go env GOPATH)/pkg/mod/sigs.k8s.io
+
+# The rules to deal with builtin plugins are a bit
+# complicated because
+#
+# - Every builtin plugin is a Go plugin -
+#   meaning it gets it's own module directory
+#   (outside of the api module) with Go
+#   code in a 'main' package per Go plugin rules.
+# - kustomize locates plugins using the
+#   'apiVersion' and 'kind' fields from the
+#   plugin config file.
+# - k8s wants CamelCase in 'kind' fields'.
+# - The module name (the last name in the path)
+#   must be the lowercased 'kind' of the
```

Comment:
An extra `'`

Reply:
This comment is for the last `'` on line 167.

Reply:
good catch

Reply:
done